### PR TITLE
Ft/readme final

### PR DESCRIPTION
**Pull request by andersonresende**
Optional?
- Sentry configuration

Missing Sections?
- Production Deployment (Once it's finished)

### CODE REVIEW

File: CONTRIBUTING.md
Comment (andersonresende) [inline CONTRIBUTING.md:1]: Need to finish this page, once I have others parts done.

File: README.md
Comment (andersonresende) [inline README.md:1]: I checked the sections in our react boilerplate and in other fastapi boilerplate.

File: README.md
Comment (fjsj) [inline README.md:158]: This part of running should be separate from install. Put this under the section `Running the Application`.
Also, move `Pre-Commit Setup` to the bottom after `Testing` secion.

File: README.md
Comment (fjsj) [inline README.md:25]: I'm not sure if watchers is the best name for what we have. Perhaps call it hot-reload or something similar.

File: README.md
Comment (fjsj) [inline README.md:35]: Alembic is not mentioned here.

**Review by fjsj — CHANGES_REQUESTED** [review]
Overall looking better, but I will wait for all sections to be done to give my final approval. I may edit as well after you finish all sections.

**Comment by andersonresende** [comment]
> Overall looking better, but I will wait for all sections to be done to give my final approval. I may edit as well after you finish all sections.

Thanks, I'll keep improving it.

File: README.md
Comment (andersonresende) [inline README.md:70]: I would like to add a video or something similar to show the login/dashboard pages.

File: CONTRIBUTING.md
Comment (fjsj) [reply CONTRIBUTING.md:1]: Track this as well please

File: README.md
Comment (fjsj) [inline README.md:75]: Move this section to after Contributing. Also, tone down the wording here, as this template/boilerplate is new:
> Several people have leveraged Vinta templates and boilerplates to start projects and startups. Starting with a solid foundation allows you to create more resilient products and focus on what really matters: discovering and delivering value to your customers. If you are one of those people, we're eager to help you even more! We can spread the word about your project across our social media platforms, giving you access to a broader audience.

File: README.md
Comment (fjsj) [inline README.md:127]: This will hardly change. So say that.

File: README.md
Comment (fjsj) [inline README.md:121]: You have to explain what needs to be filled. Like: 
- Change DATABASE_URL if you're planning to use Docker to run the local servers
- Same for TEST_DATABASE_URL
- Generate SECRET_KEYs and save them (suggest how to generate, maybe with a python command like `python3 -c "import secrets; print(secrets.token_hex(32))"`)
- Set a SMTP mail server if you need

Comment (fjsj) [reply README.md:121]: Also, why don't we have a local test-only SMTP server?

File: README.md
Comment (fjsj) [reply README.md:127]: I'm seeing no gain in have two env files, local and non-local.

File: README.md
Comment (fjsj) [inline README.md:143]: Change to: "Run the project (without Docker)"
and the other section should be: "Run the project (with Docker)"

File: README.md
Comment (fjsj) [inline README.md:194]: Rename this section to "Hot Reload on development". This is the common name used for this, I checked.

**Review by fjsj — CHANGES_REQUESTED** [review]
Please check comments.

File: README.md
Comment (andersonresende) [reply README.md:115]: 1. Sure!
2. Makes sense, I'll add a local server.

Comment (andersonresende) [reply README.md:115]: Please, check if all is good now, the only things that the user needs to generate and change are the secrect keys.
I left a comment mentioning it. And there's an explanation in the .env file about the docker configuration.

File: README.md
Comment (fjsj) [inline README.md:64]: Hot-reload watchers

File: README.md
Comment (fjsj) [inline README.md:169]: I don't think we should show start-backend if the recommended way to run is in hot-reload-mode. Just show the hot-reload mode.

File: README.md
Comment (fjsj) [inline README.md:121]: User Secrets -> secret keys

File: README.md
Comment (fjsj) [inline README.md:125]: Email -> MAIL

File: README.md
Comment (fjsj) [inline README.md:131]: The cp command is wrong. There's no project_name here.
I think this should be:
```
cd fastapi_backend
cp .env.example .env
```

Also fix the frontend (see below)

**Review by fjsj — CHANGES_REQUESTED** [review]
Overall LGTM, minor improvements still needed.

File: README.md
Comment (andersonresende) [reply README.md:131]: When you use the template you have the option to set the project name. But, I can just change as you suggested.

File: README.md
Comment (andersonresende) [reply README.md:169]: Now, the command is watch-backend and watch-frontend. What command name do you suggest?

Comment (andersonresende) [reply README.md:169]: - make backend-hotreload
- make hot-backend
- make backend-live
- make backend-run-live

Comment (fjsj) [reply README.md:169]: There's a start-backend in the README. Check it.

Comment (fjsj) [reply README.md:169]: What I mean is: we should only recommend watch mode on readme. Not  start-backend.

**Review by fjsj — APPROVED** [review]

**Pull request merged**
